dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Voltage/TCL1.mo

```diff
@@ -0,0 +1,46 @@
+within Dynawo.Electrical.Controls.Voltage;
+
+/*
+* Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+*/
+
+model TCL1 "Tap Changer Lock (TCL)"
+  /* Lock tap changers when the voltage level goes below a predefined threshold
+     in order to avoid a voltage collapse */
```

Comment:
Adapt the comment to highlight the fact that one and only one voltage is controlled. Something like this:

model TapChangerBlocking1 "Tap Changer Blocking special protection scheme monitoring the voltage on one bus"

Reply:
done